### ISSUE TITLE
Pep8 Name Formatting

### ISSUE THREAD

**Issue by cadop** [cadop/pyCGM#4]
At least try to do some pep8 formatting on functions.

**Comment by AdityaJ7** [comment]
Hi would like to work on this issue, should I add flake8 in github actions to check PEP8 formatting as well?

**Comment by cadop** [comment]
Hi @AdityaJ7  there are some new members on the project which was why I started making these issues. Probably will need to coordinate and break these up a bit more.  However if you want to pickup the github actions first using flake8 that would be good. 

https://github.com/cadop/pyCGM/issues/6

**Comment by AdityaJ7** [comment]
> Hi @AdityaJ7 there are some new members on the project which was why I started making these issues. Probably will need to coordinate and break these up a bit more. However if you want to pickup the github actions first using flake8 that would be good.
> 
> #6

Yeah I can make a PR with flake8 linting added to GitHub actions if you want.

**Comment by AdityaJ7** [comment]
@cadop Just a small question, are you planning to remove Python 2.7 support?

**Comment by cadop** [comment]
My hope is to keep it compatible as long as possible.  Even though its dead, there are some projects that still use this repo that were built in it, so not breaking it would be nice.

**Comment by AdityaJ7** [comment]
> My hope is to keep it compatible as long as possible. Even though its dead, there are some projects that still use this repo that were built in it, so not breaking it would be nice.
> 
> Can you comment on #6 , I will assign to you

Ok, I just did that
